Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -2,72 +2,64 @@
+For this, you just need to create a `.github/CODEOWNERS` file in your repo with the following content (if you have a `CODEOWNERS` file already at the root of your repository, we advise to move it inside a `.github/` directory to clean up your repo root).
 
-But if you have created new teams (team listing reviewers + proxy team) you'll need to log in to https://pullreminders.com and add the GitHub teams to Pull Assigner's configuration via [this screen](https://pullreminders.com/installs/6124714/assigner)
+```
+# Global rule for the whole codebase.
+#  - The empty `iOS-PullAssigner` team will act as a proxy for the PullAssigner bot.
+#    When a new PR arrives, PullAssigner will be triggered, which will then pick members from the iOS-Admin team
 
- - Click "Add Team"
- - Select the real (not proxy) team you've created before in the dropdown menu
- - Configure the team in the next screen, especially the number of reviewers to assign, the algorithm, but also the proxy team you previously created, and select to delete that (proxy) team review request after assigning reviewers (†).
+* @babylonhealth/iOS-Devs
+```
 
- (†) Note that if your repository's protected branch is configured with "Require review from Code Owners" checked, then any member or team mentioned in your `CODEOWNERS` file will stay assigned as reviewer. Which means that in that case, your proxy team, being typically mentioned in your `CODEOWNERS` to always be auto-assigned, won't be removed from your PR even if you chose "Delete after assigning reviewer(s)" for the "Team review request" setting in your https://pullreminders.com setup. It will only be able to be deleted if the "Require review from Code Owners" setting is unchecked in your repo.
+## Enabling GitHub's code review assignment
 
-## Update GitHub's PullAssigner app settings
+Once the team and `CODEOWNERS` file has been created, we can go ahead and enable GitHub's reviewer assignment feature.
 
-⚠️ This step will require the assistance of someone from `#devops` who has access to the GitHub's *organization* settings.
+1. Head to the organisation's [home page](https://github.com/babylonhealth).
+2. Go to the "Teams" tab and search for your team ([e.g. here for `iOS-Devs`](https://github.com/orgs/babylonhealth/teams/ios-devs)).
+3. Go to the "Settings" page (to access / edit your team's settings you'll need to have the `Maintainer` team role).
+4. In "Settings" go to "Code review assignment".
+5. Tick "Enable auto assignment" and configure the settings as desired. 
 
-Ask `#devops` to go to the GitHub organization's settings and under the "Installed GitHub Pages" [here](https://github.com/organizations/babylonhealth/settings/installations).
+**Note:** You'll probably want to tick "If assigning team members, don't notify the entire team" to prevent the entire team getting notifications for every PR, even if they haven't been individually assigned to it.
```

Comment:
```suggestion
**Note:** You'll probably want to tick "If assigning team members, don't notify the entire team" to prevent the entire team from getting notifications for every PR, even if they haven't been individually assigned to it.
```